test/integration/http-service.yaml.tmpl

```diff
@@ -43,8 +43,6 @@ spec:
           - "{{.port1}}"
           - --port
           - "{{.port2}}"
-          - --version
```

Comment:
I think if you remove this, the server will throw an error.. Its harmless anyway. Its used by routing tests.